### ISSUE TITLE
Double dropdown icons in IE9

### ISSUE THREAD

**Issue by mod-jeremyredding** [Financial-Times/o-forms#98]
After updating to version 1.0.2, I am seeing the native drop down icon as well as the custom icon in IE9.

This was not happening in 1.0.0.

![image](https://cloud.githubusercontent.com/assets/8138004/8336680/69e42cf8-1a63-11e5-9998-7a12e6719e14.png)

**Comment by kaelig** [comment]
Thanks, is this happening locally or using the build service?

If it happens locally only, it could be related to LibSass.

**Comment by mod-jeremyredding** [comment]
It happens when using a css file generated through the build service.

**Comment by mod-jeremyredding** [comment]
It may not be an issue with the specific version. I tried rebuilding the 1.0.0 through the build service and the IE browser hacks are not there either. 

When version 1.0.0 was built in April, it contained IE browser hacks like the ones below:

.o-forms-select {
    cursor:pointer;
    background-image:url(data:image/svg+xml;base64,{OMITTED});
    background-position:right center;
    background-position:right 5px center;
    background-repeat:no-repeat;
    background-origin:border-box;
    background-size:19px;
    padding-right:24px;
    padding-top:6px;
    line-height:110%;
    padding-bottom:6px\9;
    padding-right:5px\9;
    background-image:none\9;
    text-indent:.01px;
    text-overflow:
}

I am not seeing them in the built CSS today:

**Comment by kaelig** [comment]
Ok, this is a bit strange, as I thought I had fixed these issues a while ago.

Thanks, I'll investigate.

**Comment by kaelig** [comment]
Thanks again for reporting this.

I've investigated the bug and I remembered that this is something I decided not to fix because of a lack of implement browser hacks to target IE9 only or IE <= 9.

If removing the double arrow is important for the usability of your product, I suggest you use conditional comments, for example like this:

```html
<!--[if IE 9]>
<style type="text/css">
.o-forms-select {
	padding-right: 5px;
	background-image: none;
}
</style>
<![endif]-->
```

Let me know if that fixes the issue.
